plugin/trino-hive/src/main/java/io/trino/plugin/hive/HiveColumnProperties.java

```diff
@@ -14,7 +14,7 @@
 package io.trino.plugin.hive;
 
 import com.google.common.collect.ImmutableList;
-import io.trino.plugin.hive.aws.athena.projection.ProjectionType;
```

Comment:
We could move this to a top level `io.trino.plugin.hive.projection` package. The two classes in `aws` need to be moved to the `metastore.glue` package anyway.